packages/core/src/browser/dialogs.ts

```diff
@@ -23,6 +23,18 @@ import { FrontendApplicationContribution } from './frontend-application';
 @injectable()
 export class DialogProps {
     readonly title: string;
+    /**
+     * Determines the maximum width of the dialog in pixels.
+     */
+    maxWidth?: number;
```

Comment:
What is the default? The same for wordWrap.

Reply:
The dafault value for maxWidth and wordWrap is 'undefined'

Reply:
ok, thx